rdf-abac-fuseki/src/main/java/io/telicent/jena/abac/fuseki/server/BearerMode.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.telicent.jena.abac.fuseki.server;
+
+public enum BearerMode { REQUIRED, OPTIONAL/*, NONE*/ }
```

Comment:
We only ever seem to use REQUIRED (at least in this repo) so I don't know why we're commenting out NONE

Reply:
This code is taken from Apache Jena (which has the same coment). 
This copy can be dropped now Jena 5.1.0 is available.

OPTIONAL and NONE are different. OPTIONAL processes bearer auth if available but does not require it. NONE (if implemented) is "reject if any bearer auth supplied" i.e. to check that no false impression is created.